.busted

```diff
@@ -0,0 +1,9 @@
+return {
```

Comment:
I just stole this from APIcast :) https://github.com/3scale/apicast/blob/master/.busted
No idea how it works, to be honest.